tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -132,11 +132,11 @@ export type BuckWebSocketMessage = {
   } | { type: \"BuildProgressUpdated\", progressValue: number } | { type: \"BuildFinished\", exitCode: number } | { type: \"BuildStarted\" } | { type: \"ParseStarted\" } | { type: \"ParseFinished\" } | { type: \"RunStarted\" } | { type: \"RunComplete\" };
 
 // Missing one level of indentation because of the comment
-const rootEpic = (actions, store) => // Log errors and continue.
-combineEpics(...epics)(actions, store).catch((err, stream) => {
-  getLogger().error(err);
-  return stream;
-});
+const rootEpic = (actions, store) =>
+  combineEpics(...epics)(actions, store).catch((err, stream) => {
```

Comment:
The comment is gone because it's a leading comment and is attached to the call expression chain, and you don't call `print` properly around the nodes anymore. Comments are automatically added in the `print` function by adding the leading and trailing ones around the printed node.

I'm not sure what the right fix is, I'm still studying your code. If you can, try to call `print` on a node instead of dissecting it and printing parts of it.